test/test_profiler_utils_golden.py

```diff
@@ -0,0 +1,2502 @@
+# Owner(s): ["oncall: profiler"]
```

Comment:
It seems to me that this would be a lot clearer as just a json and a function to regenerate it in `test_profiler.py`.